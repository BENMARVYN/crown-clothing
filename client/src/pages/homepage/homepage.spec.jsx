import React from "react";
import { shallow } from "enzyme";
import HomePage from "./homepage.component";

describe("Testinf HomePage component", () => {
  it("Should be match snapshot", () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
