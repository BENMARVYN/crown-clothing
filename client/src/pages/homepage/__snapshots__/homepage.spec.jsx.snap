// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testinf HomePage component Should be match snapshot 1`] = `ShallowWrapper {}`;
